### ISSUE TITLE
[Source IBM Db2] - Add SSH tunnel connectivity

### ISSUE THREAD

**Issue by philippeboyd** [airbytehq/airbyte#16837]
## Tell us about the problem you're trying to solve
Unable to connect to a DB2 instance behind a private network.

## Describe the solution you’d like
Same MSSQL, and MySQL , add the option to connect to a DB2 instance using an SSH tunnel

**Comment by philippeboyd** [comment]
I've seen the following in the the Docs:

> Note. Unfortunately SSL and SSH are incompatible because SSH changes the visible name of the target.

Not sure I fully understand that note, isn't it an SSH port forwarding after all? Why would SSH change the visible name of the target?

**Comment by marcosmarxm** [comment]
@andriikorotkov could you help here?

**Comment by andriikorotkov** [comment]
@philippeboyd Yes, that is a false note. It's good that you noticed it. I removed it so as not to mislead users. Thank you.

**Comment by philippeboyd** [comment]
@marcosmarxm @andriikorotkov any news on this feature? Do you guys know if it's being worked internally at Airbyte?

**Comment by philippeboyd** [comment]
As Thanos once said:
> Fine I'll do it myself.

I'm currently implementing and testing the SSH tunnelling option for DB2, which will also close #18470

**Comment by octavia-squidington-iii** [comment]
At Airbyte, we seek to be clear about the project priorities and roadmap. This issue has not had any activity for 365 days, suggesting that it's not as critical as others. It's possible it has already been fixed. It is being marked as stale and will be closed in 20 days if there is no activity. To keep it open, please comment to let us know why it is important to you and if it is still reproducible on recent versions of Airbyte.
